src/components/Hero.astro

```diff
@@ -7,6 +7,7 @@ const { siteState } = Astro.props; // for updating content depending on the stag
 let title = "Revolution".toUpperCase();
 let date = "    February 24-25    ".toUpperCase(); // Keep the space in order to match with the RevolutionUC Text
```

Comment:
Can you hide this for the livesite? We don't need the date when the hackathon is live.